lampod/src/ln/inventory_manager.rs

```diff
@@ -39,6 +40,18 @@ impl InventoryHandler for LampoInventoryManager {
                 let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
                 let blockheight = height.unwrap_or_default();
                 let lampo_dir = self.channel_manager.conf.root_path.to_string();
+                // We provide a vector here as there may be other types of address in future like tor and ipv6.
+                let address = self
+                    .channel_manager
+                    .conf
+                    .announce_addr
+                    .clone()
+                    .unwrap_or("127.0.0.1".to_string());
```

Comment:
```suggestion
```

an address can be `None`, do not make any assumption on people address

Reply:
If the address is not specified inside the `conf`. We default it to `127.0.0.1` as mentioned inside `peer_menager` :

https://github.com/vincenzopalazzo/lampo.rs/blob/1f92659e1551d8bd94fdea3092f1a0b37ef24524/lampod/src/ln/peer_manager.rs#L139

Reply:
@Harshit933 also in this case it is wrong, we should set it to `None` and avoid to bind the server on a specific peer.

However it is unrelated to this PR

Reply:
Opening this as an new issue